<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>711</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>145</number>
    </attribute>
    <column>
     <property name="text">
      <string>Seller Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CNIC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 70px; 
}</string>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>500</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 70px; 
}</string>
    </property>
    <property name="text">
     <string>Deactivate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>114</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 70px; 
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
